l10n/en/ui.ftl

```diff
@@ -20,3 +20,6 @@ ui-show-all = Show All
 ui-hide-all = Hide All
 ui-learn-more = Learn more
 ui-view = View
+
+# An accessible label used to describe the purpose of a page "banner".
+ui-banner-label = Banner
```

Comment:
"Banner" sounds perhaps too much like a 3rdparty advertising… I'm trying to come up with a way to better describe this kind of content from the page outline perspective… Any ideas? (I can only think of "Promotion" or something similar, but that's not much better FWIW…)